src/tag/tag.tsx

```diff
@@ -0,0 +1,37 @@
+import { TagProps } from "./types";
+import { Label, Wrapper } from "./tag.style";
+
+export const Tag = ({
+    type,
+    colorType = "black",
+    children,
+    interactive = false,
+    icon,
+    onClick,
+    ...otherProps
+}: TagProps): JSX.Element => {
+    // =============================================================================
+    // EVENT HANDLERS
+    // =============================================================================
+    const handleClick = (
```

Comment:
since there's no special handling for click events, you can omit it from the destructuring and pass it down as part of `otherProps`